readalongs/api.py

```diff
@@ -0,0 +1,146 @@
+"""
+api.py: API for calling readalongs CLI commands programmatically
+
+In this API, functions take the same arguments as on the readalongs
+command-line interface. The mapping between CLI options and API options is
+that the first long variant of an option described in "readalongs cmd -h" is
+the API option name, with hyphens replaced by undercores.
+
+Example from readalongs align -h:
+    option in CLI                       option in API
+    ================================    =================================
+    -l, --language, --languages TEXT    language=["l1", "l2"]
+    -f, --force-overwrite               force_overwrite=True
+    -c, --config PATH                   config=os.path.join("some", "path", "config.json")
+                                     OR config=pathlib.Path("/some/path/config.json")
+
+As shown above, file names can be constructed using os.path.join() or a Path
+class like pathlib.Path. Warning: don't just use "/some/path/config.json"
```

Comment:
I like this - you don't usually see this level of documentation - it's just assumed to be understood, but given that we might be dealing with novice programmers, this is helpful.